### PR TITLE
Threshold now is expressed in terms of a percentage of the voters

### PR DESCRIPTION
**Pull request by erwinvanthiel**

### CODE REVIEW

File: common/src/main/java/nl/tudelft/trustchain/common/util/VotingHelper.kt [hunk 1]
```diff
@@ -194,7 +195,7 @@ class VotingHelper(
     /**
      * Return the percentage of required votes or -1 when a threshold is not made but all votes are received.
      */
-    fun getVoteProgressStatus(block: TrustChainBlock, threshold: Int = -1): Int {
+    fun getVoteProgressStatus(block: TrustChainBlock, thresholdPercentage: Int = -1): Int {
```
Comment (emieldesmidt) [inline common/src/main/java/nl/tudelft/trustchain/common/util/VotingHelper.kt:198]: I would prefer the `thresholdPercentage` being a value of the community, and that this method can figure out from the block what type of vote it is, and return an appropriate value

Comment (emieldesmidt) [reply common/src/main/java/nl/tudelft/trustchain/common/util/VotingHelper.kt:198]: Such that it is not something that should be passed in the function arguments

Comment (PraveshMoelchand) [reply common/src/main/java/nl/tudelft/trustchain/common/util/VotingHelper.kt:198]: I don't see how your comment about the community relates to this method.

Comment (PraveshMoelchand) [reply common/src/main/java/nl/tudelft/trustchain/common/util/VotingHelper.kt:198]: But I think this method works fine

**Pull request merged**